src/components/panels.tsx

```diff
@@ -52,40 +53,55 @@ function Root({ children }: React.PropsWithChildren) {
   })
 
   useEvent("keydown", (event) => {
-    // Focus prev/next panel with `command + shift + left/right`
-    if (
-      (event.key === "ArrowLeft" || event.key === "ArrowRight") &&
-      event.metaKey &&
-      event.shiftKey
-    ) {
-      const panelElements = Array.from(document.querySelectorAll<HTMLElement>("[data-panel]"))
-
-      const focusedPanelElement = panelElements.find((panelElement) =>
-        panelElement.contains(document.activeElement),
-      )
-
-      const focusedPanelIndex = focusedPanelElement
-        ? panelElements.indexOf(focusedPanelElement)
-        : -1
-
-      if (event.key === "ArrowLeft" && event.altKey) {
-        const firstPanel = panelElements[0]
+    // Focus prev/next panel with `option + [shift] + left/right`
+    const getPanelElements = () =>
+      Array.from(document.querySelectorAll<HTMLElement>("[data-panel]"))
+    const getPanelsAndFocusedPanelIndex = () => {
+      const panelElements = getPanelElements()
+      return {
+        panelElements,
+        focusedPanelIndex: panelElements.findIndex((panelElement) =>
+          panelElement.contains(document.activeElement),
+        ),
+      }
+    }
```

Comment:
Nitpick: The function abstraction doesn't feel quite right here. What if we made `getFocusedPanelIndex` a function of `panelElements`:

```suggestion
    const getFocusedPanelIndex = (panelElements: Array<HTMLElement>) => {
      return panelElements.findIndex((panelElement) => panelElement.contains(document.activeElement)) 
    }
```